pyanaconda/bootloader.py

```diff
@@ -1861,12 +1861,18 @@ def packages(self):
             super(EFIGRUB, self).packages
 
 class Aarch64EFIGRUB(EFIGRUB):
-    _packages64 = ["grub2-efi-aa64", "shim-aa64"]
     _serial_consoles = ["ttyAMA", "ttyS"]
     _efi_binary = "\\shimaa64.efi"
 
+    def __init__(self):
+        super(Aarch64EFIGRUB, self).__init__()
+        self._packages64 = ["grub2-efi-aa64", "shim-aa64"]
+
 class MacEFIGRUB(EFIGRUB):
-    _packages64 = [ "grub2-tools-efi", "mactel-boot" ]
+    def __init__(self):
+        super(MacEFIGRUB, self).__init__()
+        self._packages64.extend(["grub2-tools-efi", "mactel-boot"])
```

Comment:
I really don't like this patch for alpha branch it is making instance variables from class variables so it is bug prone. I would like to see a clear solution to master and here. However, yeah it's already there so we can't do much about it here.